index.html

```diff
@@ -76,7 +76,12 @@ <h2>Seccion 4 del blob</h2>
                         <img src="imagen/dev f.png" alt="proyecto del curso de React Native">
                     </section>
                 </article>
-
+                <article>
+                    <section>
+                        <h1> Hacktoberfest Monterrey </h1>
+                        <p> Esto es Hacktoberfest! </p>
```

Comment:
```suggestion
                        <p> Esto es Hacktoberfest 2019! </p>
```

Hay que agregar el year

Reply:
Okay